dal/dbConn.py

```diff
@@ -0,0 +1,28 @@
+from tinydb import TinyDB, Query
+
+db = TinyDB('db.json')
+
+
+def db_insert(name) -> None:
+    """
+        Inserts a workspace with id, name and wid to the db
+        :param name: name of the workspace to be inserted in db
+    """
+    count = len(db)
+    if count == 0:
+        identity = 1
+    else:
+        db_list = db.all()
+        db_dict = db_list[count-1]
```

Comment:
@mansikulkarni96 This line does not necessarily give the latest record on the db. You might have to do a sort based on the id attribute then fetch the latest one.

Reply:
Will be adding uuid instead of auto incremented id.